CMakeLists.txt

```diff
@@ -189,7 +188,6 @@ include_directories(SYSTEM third-party/inja)
 # Enable SSL Support, most URLs now-a-days use SSL!
 # TODO - probably integrate with ZSTD since we have it already
 set(CURL_USE_LIBSSH2 OFF)
-set(CURL_ZLIB OFF)
```

Comment:
curious if you found a reason this wasn't needed, I unfortunately can't remember why i added it -- but i know it was when i was troubleshooting why the binaries were expecting some libraries to be dynamically linked

Reply:
I was getting linker errors with this option set. But I can't reproduce them now :thinking: